amplipi/tests.py

```diff
@@ -62,37 +64,40 @@ def available(self) -> bool:
     except:
       return False
 
-RCA_INPUTS = { sid: 996 + sid for sid in range(models.MAX_SOURCES) }
+
+RCA_INPUTS = {sid: 996 + sid for sid in range(models.MAX_SOURCES)}
```

Comment:
suggestion: it may (or may not) make more sense to `from amplipi.defaults import RCAs` and use that instead of this.

Reply:
I'll take a look at that. I don't think that existed when I wrote this. This change is only popping up here because of auto-format actually kicking in now that I fixed the things VS Code has been yelling at us about for ~weeks~ ~months~ years.

Reply:
it definitely did not exist before a month ago even. you could definitely leave it for future refactor, np.